src/app/components/notification-box/notification-box.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="notification-box" *ngIf="isVisible">
+    <mat-progress-bar mode="determinate" [value]="progress"></mat-progress-bar>
+    <div style="display: flex; height: 80px;">
+        <div class="edgeline"></div>
+        <div class="content">
+            <app-adbutler theme="dark"></app-adbutler>
+            <mat-icon (click)="closeNotification()" class="close-button">close</mat-icon>
+            <!-- <button class="check-button" mat-button>CHECK IT OUT</button> -->
+        </div>
+        <div class="edgeline"></div>
```

Comment:
I'm wondering if in .content { border-left/right/bottom } wouldn't be sufficient.

Reply:
Fixed